modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1214 @@
+
+* xref:new-premium-plugin[New Premium Plugin]
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-plugin-end-of-life-announcement[Accompanying Premium Plugin end-of-life announcement]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvement[Improvement]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+* xref:known-issues[Known issues]
+// NB: edit the equivalent section of nav.adoc when the remove or make visible decision is made.
+
+NOTE: Although there was a Community release of {productname} 6.4, it was not released as a {enterpriseversion} or {cloudname} release. The first commercially supported release of {productname} post version 6.3.2 is {productname} 6.4.1.
+
+[[new-premium-plugin]]
+== New Premium Plugin
+
+=== Advanced Template 1.1.0
+
+The {productname} 6.4.1 release includes the release of a new premium plugin: **Advanced Template**.
+
+The **Advanced Template** Premium plugin enables the creation of complex, interactive templates using {productname}.
+
+Templates are declared as `const` variables that can be setup as part of a {productname} instance.
+
+They can also be added to a {productname} instance by end-users selecting content in a {productname} document and saving the selection as a template via the {productname} user-interface.
+
+For information on the **Advanced Template** plugin, see: xref:advanced-templates.adoc[Advanced Template].
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.4.1.
+
+=== Advanced Code Editor 3.2.1
+
+The {productname} 6.4.1 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.2.1 includes the following improvements and additions.
+
+==== New button to copy the current dialog content.
+
+This update adds a new **Copy code** button to the Advanced Code Editor dialog, enabling the copying of the entire contents of the dialog without having to first select.
+
+==== New buttons to increase and decrease the font size in the dialog.
+
+This update adds two new buttons for increasing and decreasing the font size of code displayed in the Advanced Code Editor dialogs.
+
+==== New button to toggle dark mode.
+
+This update adds a new **Dark/light mode** button for toggling the Advanced Code Editor dialog between dark and light display modes.
+
+==== New button to toggle fullscreen.
+
+This update adds a new **Fullscreen** button that for toggling the Advanced Code Editor dialog between fullscreen and dialog modes.
+
+NOTE: For the Advanced Code Editor to offer a full-screen mode requires the xref:fullscreen.adoc[Full screen] plugin and requires {pluginname} to be running in xref:advcode.adoc#advcode_inline[inline mode].
```

Comment:
```suggestion
==== New button to copy the current Code Editor content.

This update adds a new **Copy code** button to the Advanced Code Editor Code Editor, enabling the copying of the entire contents of the Code Editor without having to first select.

==== New buttons to increase and decrease the font size in the Code Editor.

This update adds two new buttons for increasing and decreasing the font size of code displayed in the Advanced Code Editor Code Editors.

==== New button to toggle dark mode.

This update adds a new **Dark/light mode** button for toggling the Advanced Code Editor Code Editor between dark and light display modes.

==== New button to toggle fullscreen.

This update adds a new **Fullscreen** button that for toggling the Advanced Code Editor Code Editor between fullscreen and Code Editor initial size.

NOTE: For the Advanced Code Editor to offer a full-screen mode requires the xref:fullscreen.adoc[Full screen] plugin and requires {pluginname} to be running in xref:advcode.adoc#advcode_inline[inline mode].

NOTE: For the Advanced Code Editor to offer a Copy button requires a https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts[*secure contexts*]
```

If something sounds bad in English feel free to change it, I just removed the term `dialog` and used `Code Editor` to refer to both dialog and inline mode

This is particularly important for the fullscreen mode which can be used only in inline mode.

I also add a second `NOTE` if it is not correct to have 2 notes in row, feel free to merge them

Reply:
```suggestion
=== Advanced Code Editor 3.2.1

The {productname} 6.4.1 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.2.1 includes the following improvements and additions.

==== New button to copy the current dialog content.

This update adds a new **Copy code** button to the **Advanced Code Editor** for editors using either `advcode_inline`, `inline` or using the default dialog within {productname}'s `Classic mode`. Its worth noting that both options allow the user to copy the entire contents of from the `source code` without having to first select the content.

==== New buttons to increase and decrease the font size in the dialog.

This update adds two new buttons for increasing and decreasing the font size of code displayed in the Advanced Code Editor.

==== New button to toggle dark mode.

This update adds a new **Dark/light mode** button that can be used within the Advanced Code Editor to toggle between either dark or light display modes.

==== New button to toggle fullscreen.

This update adds a new **Fullscreen** button that can be used within the Advanced Code Editor to toggle between fullscreen and dialog modes.

NOTE: For the Advanced Code Editor to offer a full-screen mode, it requires the xref:fullscreen.adoc[Full screen] plugin and requires {pluginname} to be running in xref:advcode.adoc#advcode_inline[inline mode].

NOTE: For the Advanced Code Editor to offer a **Copy** button, it requires a https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts[*secure contexts*]
```

Reply:
Discussed with @tiny-ben-tran for approval